nixos/modules/virtualisation/containers.nix

```diff
@@ -596,6 +596,8 @@ in
                   { config, pkgs, ... }:
                   { services.postgresql.enable = true;
                     services.postgresql.package = pkgs.postgresql96;
+                    
+                    system.stateVersion = "17.03";
```

Comment:
Perhaps could try having `system.stateVersion = "${config.system.nixosRelease}";` here? My idea is to automatically keep it up to date…

Reply:
yeah, that would be great, but if we go that way, we should go consistently.

See https://github.com/NixOS/nixpkgs/blob/release-17.09/nixos/modules/misc/version.nix#L53, for example. It backreferences 16.03